waspc/src/Wasp/AI/GenerateNewProject/WaspFile.hs

```diff
@@ -89,12 +89,20 @@ fixWaspFile newProjectDetails waspFilePath plan = do
             ${compileErrorsText}
 
             Some common mistakes to look for:
+              - using non-default imports in page components 
+                  In a Wasp page, the component should always use the "default import" JS syntax.
+                  Instead of `component: import { PageName } from ...`, it should 
+                  always be `component: import PageName from ...`.
+                  Fix these by identifying named imports in wasp pages, and replacing 
+                  them with default imports. This only relates to wasp pages, other parts 
+                  of a Wasp file do not have to use the default imports.
```

Comment:
```suggestion
                  In a Wasp `page` declaration, the `component` should always use the "default import" JS syntax.
                  Instead of `component: import { PageName } from ...`, it should 
                  always be `component: import PageName from ...`.
                  Fix these by identifying named imports in wasp pages, and replacing 
                  them with default imports. This only relates to wasp pages, other parts 
                  of a Wasp file do not have to use the default imports.
```

Reply:
Again, this change probably does not matter from the perspective of the LLMs, but it makes sense from a human perspective.